WalletWasabi.Fluent/Controls/PrivacyTextPresenter.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using System.Reactive.Linq;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Media;
+using Avalonia.Utilities;
+using WalletWasabi.Fluent.ViewModels;
+
+namespace WalletWasabi.Fluent.Controls;
+
+public class PrivacyTextPresenter : UserControl
```

Comment:
For that I had in mind using something like that https://github.com/zkSNACKs/WalletWasabi/blob/5bf14886691cff522962e3c72d277cd5bb51c536/WalletWasabi.Fluent/TreeDataGrid/TreeDataGridPrivacyTextCell.cs#L13

Reply:
@wieslawsoltes I tried to adapt that code, but I couldn't make it work. It would hang the entire UI, so I was obviously doing something wrong.